CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Removed usages of Random in unit tests to ensure the tests are deterministic.
 * **validate** will now run on all the pack content items when the pack supported marketplaces are modified.
 * Fixed an issue where errors in **validate** were logged as `info`.
+* Fixed an issue where parameters of type 9 could not be hidden in specific marketplaces.
```

Comment:
```suggestion
* Fixed an issue where parameters of type 9 could not be hidden in specific marketplaces.
```

It's a feature, not a bug. Please use the same sentence we used when adding the _hide marketplaces_ in type-4, and mention it's now available for credentials-type (type 9) parameters.